src/main/java/ua/su/repository/impl/ClinicRepositoryImpl.java

```diff
@@ -0,0 +1,71 @@
+package ua.su.repository.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import ua.su.domain.Clinic;
+import ua.su.repository.ClinicRepository;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+
+@Repository
+public class ClinicRepositoryImpl implements ClinicRepository {
+
+    private static final BeanPropertyRowMapper<Clinic> ROW_MAPPER =
+            new BeanPropertyRowMapper<>(Clinic.class);
+
+    private final JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public ClinicRepositoryImpl(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public List<Clinic> findAll() {
+        return jdbcTemplate.query("SELECT * FROM clinics;", ROW_MAPPER);
+    }
+
+    @Override
+    public Clinic getOne(Long id) {
+        return jdbcTemplate.queryForObject("SELECT * FROM clinics WHERE id = ?", ROW_MAPPER, id);
+    }
+
+    @Override
+    public Clinic insert(Clinic clinic) {
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+        jdbcTemplate.update(connection -> {
+            PreparedStatement ps =
+                    connection.prepareStatement("insert into clinics(name, address, phone, is_insurance_supported, " +
```

Comment:
Here sql is in a lower case, but in the getOne() method is upper case. It is better to use same sql everywhere.

Reply:
I modified SQL statements to the upper case.